Makefile.uk

```diff
@@ -21,16 +21,18 @@ CINCLUDES    += -I$(CONFIG_UK_BASE)/include
 CXXINCLUDES  += -I$(CONFIG_UK_BASE)/include
 GOCINCLUDES  += -I$(CONFIG_UK_BASE)/include
 
-RUSTCFLAGS-y	+= --emit=obj --crate-type=rlib --edition=2018 \
+RUSTCFLAGS-y	+= +nightly --emit=obj --crate-type=rlib --edition=2018 \
```

Comment:
The "+<toolchain>" syntax is rustup specific and IIRC does not work with a normal rust installation (from the package manager or a manual installation). It also requires installing the toolchain manually (The error message does not seem to give instructions how to do that).
As an alternative there are [rust-toolchain](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file) files, which rustup will use to automatically install the required rust version.

Reply:
Thanks didn't know that, will remove that then